holidays/countries/india.py

```
@@ -497,29 +545,154 @@ def _populate_optional_holidays(self):
+        self._add_holiday_dec_10(tr("Shaheedi Divas"))
+
+        # Hindu holidays.
+
+        # Wangala Festival.
+        self._add_wangala_festival(tr("Wangala Festival"))
 
     # Bihar.
     def _populate_subdiv_br_public_holidays(self):
+        pass
+
+    def _populate_subdiv_br_government_holidays(self):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_populate_subdiv_br_government_holidays`

Add return type annotation: `None`

(ANN202)